vital/types/sfm_constraints.cxx

```diff
@@ -230,8 +230,9 @@ ::get_camera_orientation_prior_local(frame_id_t fid,
       return false;
     }
 
-    R_loc = compose_rotations<double>(platform_heading, platform_pitch, platform_roll,
```

Comment:
I'm curious if leaving this compose rotations function would be useful, even if it gets renamed to something more precise. Writing this boilerplate seems like it could be prone to errors.